### ISSUE TITLE
Isn't Mexc open for use?

### ISSUE THREAD

**Issue by halocdn** [ccxt/ccxt#23775]
### Programming Languages

PHP

### CCXT Version

1.16

### Description

Supported exchanges: _1broker, _1btcxe, acx, allcoin, anxpro, anybits, bibox, binance, bit2c, bitbank, bitbay, bitfinex, bitfinex2, bitflyer, bithumb, bitkk, bitlish, bitmarket, bitmex, bitsane, bitso, bitstamp, bitstamp1, bittrex, bitz, bl3p, bleutrade, braziliex, btcbox, btcchina, btcexchange, btcmarkets, btctradeim, btctradeua, btcturk, btcx, bxinth, ccex, cex, chbtc, chilebit, cobinhood, coinbase, coinbasepro, coincheck, coinegg, coinex, coinexchange, coinfalcon, coinfloor, coingi, coinmarketcap, coinmate, coinnest, coinone, coinsecure, coinspot, cointiger, coolcoin, crypton, cryptopia, deribit, dsx, ethfinex, exmo, exx, fcoin, flowbtc, foxbit, fybse, fybsg, gatecoin, gateio, gdax, gemini, getbtc, hadax, hitbtc, hitbtc2, huobi, huobicny, huobipro, ice3x, independentreserve, indodax, itbit, jubi, kraken, kucoin, kuna, lakebtc, lbank, liqui, livecoin, luno, lykke, mercado, mixcoins, negociecoins, nova, okcoincny, okcoinusd, okex, paymium, poloniex, qryptos, quadrigacx, quoinex, southxchange, surbitcoin, therock, tidebit, tidex, urdubit, vaultoro, vbtc, virwox, wex, xbtce, yobit, yunbi, zaif, zb

### Code

```
  

```

**Comment by sc0Vu** [comment]
@halocdn Yes, you can find it in the README.md https://github.com/ccxt/ccxt

**Comment by halocdn** [comment]
![Uploading Ekran Resmi 2024-09-21 22.32.07.png…]()

**Comment by carlosmiei** [comment]
Hello @halocdn your version is really outdated, can you please upgrade and try again?

**Comment by carlotestor** [comment]
MEXC **is** supported in modern CCXT. Composer `ccxt/ccxt ^1.16` is many years out of date (pre-dates current MEXC ids).

```bash
composer require ccxt/ccxt:^4
```

```php
$mexc = new \ccxt\mexc();
$markets = $mexc->load_markets();
echo $mexc->id; // mexc
```

After upgrading, `mexc` appears in `\ccxt\Exchange::$exchanges`. Please upgrade and reopen only if something still fails on v4. Safe to close the “isn’t MEXC open?” question.
